README.md

```diff
@@ -98,8 +108,6 @@ For further information or help, don't hesitate to get in touch on the [Slack `#
 <!-- TODO nf-core: Add citation for pipeline after first release. Uncomment lines below and update Zenodo doi and badge at the top of this file. -->
 <!-- If you use nf-core/mcmicro for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX) -->
 
-<!-- TODO nf-core: Add bibliography of tools and data used in your pipeline -->
```

Comment:
I removed that because it's handled by the `CITATIONS.md` file that's referenced at the end of the README.  This is how both `rnaseq` and `sarek` seem to handle it.  I did drop the ball of getting `CITATIONS.md` filled in with all the modules we use though, so I'll work on getting those added now.
Thanks so much for pointing this out.  Please let me know if you'd like me to handle it differently.
Should I also cite the tonsil data we're using?

Reply:
I've updated `CITATIONS.md` with references and URLs for the tools we use.  I used the MCMICRO reference for Backsub and MCQuant.  Please let me know if there are better papers for them.